andi/values.yaml

```diff
@@ -1,4 +1,7 @@
 global:
+  auth:
+    jwt_issuer: ""
+    auth0_domain: ""
```

Comment:
As per my comment in the other half PR: I'm not so sure that separating the Auth0 config like this makes the configuration cleaner.  I'm open to being wrong here

Reply:
I am open to either option, in this solution there will be both a local and global configuration for auth0 and implies you will be using the same instance across your microservices with different individual configurations.

As I wrote that, it sounds a bit convoluted. I am game to changing it back unless @LtChae has any oppositions